.github/workflows/ci.yml

```diff
@@ -185,18 +185,20 @@ jobs:
       matrix:
         # the arm64 build takes too long, so disable for now
         arch: [amd64]
+        image_name: [relay, relay-pop]
 
-    name: Build Docker Image (${{ matrix.arch }})
+
+    name: Build Docker Image (${{ matrix.arch }}) for ${{ matrix.image_name }}
```

Comment:
```suggestion
    name: Build Docker Image
```

Reply:
If building an image fails, I think it's helpful to know what image was trying to build. We do this for [tests](https://github.com/getsentry/relay/blob/ded63aca6aa76b42df544e238bb71d57dc02712b/.github/workflows/ci.yml#L81) on macos/windows.

Reply:
now they will look like  this
<img width="440" alt="image" src="https://github.com/getsentry/relay/assets/1931331/11696177-a8ff-413a-8584-fc36095f022e">

Reply:
Oh nice. How does it work, is it documented somewhere? I couldn't find anything relevant beyond [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#name).